api/src/controllers/referent.js

```diff
@@ -358,6 +358,42 @@ router.put("/young/:id", passport.authenticate("referent", { session: false, fai
   }
 });
 
+router.put("/young/change_cohort/:id", passport.authenticate("referent", { session: false, failWithError: true }), async (req, res) => {
+  try {
+    const { error, value } = validateYoung(req.body);
+    if (error) return res.status(400).send({ ok: false, code: ERRORS.INVALID_PARAMS, error: error.message });
+
+    const { id } = req.params;
+    const young = await YoungModel.findById(id);
+    if (!young) return res.status(404).send({ ok: false, code: ERRORS.YOUNG_NOT_FOUND });
+
+    // eslint-disable-next-line no-unused-vars
+    let { __v, ...newYoung } = value;
```

Comment:
on peut presque prendre que la cohorte
on s'en fou du reste
```suggestion
    const { cohort } = value;
```